Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -2916,18 +2920,15 @@ public virtual void UpdateFrictionCoefficient(float elapsedClockSeconds)
                 }
                 if (Simulator.WeatherType == WeatherType.Rain) // Wet weather
                 {
-                    if (Simulator.Settings.AdhesionProportionalToWeather && AdvancedAdhesionModel && !Simulator.Paused)  // Adjust clear weather for precipitation presence - base friction value will be approximately between 0.15 and 0.2
-                    // ie base value between 0.8 and 1.0 (TODO) 
-                    // note lowest friction will be for drizzle rain; friction will increase for precipitation both higher and lower than drizzle rail
+                    if (Simulator.Settings.AdhesionProportionalToWeather && AdvancedAdhesionModel && !Simulator.Paused)  // Adjust clear weather for precipitation presence
+                    // ie base value between 60% and 80% (TODO) 
+                    // note lowest friction will be for drizzle (light) rain; friction will increase for precipitation higher than drizzle rail
                     {
                         float pric = Simulator.Weather.PricipitationIntensityPPSPM2 * 1000;
-                        // precipitation will calculate a value between 0.15 (light rain) and 0.2 (heavy rain) - this will be a factor that is used to adjust the base value - assume linear value between upper and lower precipitation values
-                        if (pric >= 0.5)
-                            BaseFrictionCoefficientFactor = Math.Min((pric * 0.0078f + 0.45f), 0.8f); // should give a minimum value between 0.8 and 1.0
-                        else
-                            BaseFrictionCoefficientFactor = Math.Min((0.4539f + 1.0922f * (0.5f - pric)), 0.8f); // should give a minimum value between 0.8 and 1.0
```

Comment:
I'd prefer to keep the if clause, so there's a small transition curve from dry to wet.

```csharp
if (pric >= 0.5)
    BaseFrictionCoefficientFactor = Math.Min((pric-0.5f) * 0.0078f + 0.6f, 0.8f);
else
    BaseFrictionCoefficientFactor = 0.6f + 0.8f * (0.5f - pric);
```
![weather](https://user-images.githubusercontent.com/22003819/163538076-1ab803e5-2069-44b9-b7b3-a8f51e90a32e.jpg)

What do you think?

Reply:
Ok, I agree.

I have also looked at the code, and it only seemed to be applying this if the weather was raining or snow. I don't believe that this should be the case as it should vary from Clear - "Snow" - Rain and back again.

So I have modified the code so that it will default to the weather selected in the launch screen, but the precipitation and fog can be adjusted in game (or not as desired by the user).

I believe that the option of linking adhesion to precipitation can be removed, and will post in the forums accordingly.